exotica_examples/resources/configs/example_ik_avoid_look_at_sphere.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" ?>
+<IKSolverDemoConfig>
+
+  <SnoptIKConstrainedSolver Name="MySolver" MaxIterations="500" />
+
+  <EndPoseProblem Name="My Problem">
+
+    <PlanningScene>
+      <Scene>
+        <JointGroup>arm</JointGroup>
+        <URDF>{exotica_examples}/resources/robots/lwr_simplified.urdf</URDF>
+        <SRDF>{exotica_examples}/resources/robots/lwr_simplified.srdf</SRDF>
+        <Links>
+          <Link Name="EffPoint" Parent="lwr_arm_6_link" Transform="0 0 1"/>
+          <Link Name="LookAtTarget" Transform="-0.5 0 1"/>
+	  <Link Name="AvoidLookAtSpherePosition1" Transform="-1 0 0.5"/>
+	  <Link Name="AvoidLookAtSpherePosition2" Transform="-1 0.4 0.5"/>
+        </Links>
+      </Scene>
+    </PlanningScene>
+    
+    <Maps>
+      <LookAt Name="LookAt">
+        <EndEffector>
+          <Frame Link="EffPoint" Base="lwr_arm_6_link"/><!-- EndEffector #0 => Frame on robot to orient -->
+          <Frame Link="LookAtTarget" Base="lwr_arm_6_link"/><!-- EndEffector #1 => Target to look at, in relative frame of EffPoint -->
+          <Frame Link="LookAtTarget"/><!-- EndEffector #2 => Target to look at, in world frame (convenience frame) -->
+        </EndEffector>
+      </LookAt>
+
+      <AvoidLookAtSphere Name="AvoidLookAtSphere" UseAsCost="0" Radii="0.25 0.35" Debug="1">
+	<EndEffector>
+	  <Frame Link="AvoidLookAtSpherePosition1" Base="lwr_arm_6_link"/>
+	  <Frame Link="AvoidLookAtSpherePosition2" Base="lwr_arm_6_link"/>
+	</EndEffector>
```

Comment:
The sphere collision avoidance task map already has a convention for creating spheres.
It looks like [this in the XML](https://github.com/ipab-slmc/exotica/blob/bac70aaa560f36cd2c8f08ffdf959156dbaa0111/exotica_examples/resources/configs/example_sphere_collision.xml#L33), and [this is](https://github.com/ipab-slmc/exotica/blob/bac70aaa560f36cd2c8f08ffdf959156dbaa0111/exotations/exotica_core_task_maps/src/sphere_collision.cpp#L45) where the initializers for the spheres are loaded. You can ignore the group parameter for this task map.